engine/table/src/main/java/io/deephaven/engine/table/impl/replay/Replayer.java

```diff
@@ -13,37 +14,31 @@
 import io.deephaven.engine.table.ColumnSource;
 import io.deephaven.engine.rowset.RowSet;
 import io.deephaven.engine.updategraph.TerminalNotification;
-import io.deephaven.time.TimeProvider;
 import io.deephaven.internal.log.LoggerFactory;
 import io.deephaven.io.logger.Logger;
 
 import java.io.IOException;
+import java.time.Instant;
+import java.util.Objects;
 import java.util.TimerTask;
 import java.util.concurrent.CopyOnWriteArrayList;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.locks.Condition;
 
-import static io.deephaven.time.DateTimeUtils.millisToNanos;
-import static io.deephaven.time.DateTimeUtils.nanosToTime;
-
 /**
  * Replay historical data as simulated real-time data.
  */
 public class Replayer implements ReplayerInterface, Runnable {
     private static final Logger log = LoggerFactory.getLogger(ShiftObliviousInstrumentedListener.class);
+    private static final int MILLIS_TO_NANOS_FACTOR = 1_000_000;
```

Comment:
Maybe we want to prefer using `TimeUnit`?

Reply:
I'll delegate back to DateTimeUtils.millisToNanos etc. And remove ENABLE_MICROTIME_HACK in the process.